cde-core/resource/resources/custom/components/cgg/cgg-implementation.js

```diff
@@ -117,6 +121,7 @@ var CggComponent = UnmanagedComponent.extend({
 
 var CggDialComponent = CggComponent.extend({
   script: "system/pentaho-cdf-dd/resources/custom/components/cgg/charts/dial.js",
+  priority: 6,
```

Comment:
Don't forget the (AMD version)[https://github.com/webdetails/cde/blob/master/cde-core/resource/resources/custom/amd-components/cgg/CggDialComponent.js#L18]